program-runtime/src/loaded_programs.rs

```diff
@@ -908,6 +911,7 @@ impl<FG: ForkGraph> LoadedPrograms<FG> {
         loaded_programs_for_tx_batch: &mut LoadedProgramsForTxBatch,
     ) -> Option<(Pubkey, u64)> {
         debug_assert!(self.fork_graph.is_some());
+        let is_first_round = loaded_programs_for_tx_batch.entries.is_empty();
```

Comment:
This is technically not entirely correct as a batch can get unlucky and not hit anything in the first round, but it is for statistics only so ...

Reply:
Doesn't that mean the misses and hits will be double counted in the second time?

Reply:
I routed it from the outer loop in the bank instead. That should solve it.